docs/zh_cn/02-how-to-run/quantize_model.md

```diff
@@ -0,0 +1,65 @@
+# 部署量化模型
+
+## 为什么要量化
+
+相对于 fp32 模型，定点模型有诸多优点：
+
+* 体积更小，8-bit 模型可降低 75% 文件大小
+* 由于模型变小，Cache 命中率提升，速度更快
+* 芯片往往有对应的定点加速指令，这些指令更快、能耗更低（常见 CPU 上 int8 大约只需要 10% 能量）
+
+安装包体积、发热都是移动端评价 APP 的关键指标；而在服务端，“加速”意味着可以维持相同 QPS、增大模型换取精度提升。
+
+## mmdeploy 离线量化方案
+
+以 ncnn backend 为例，完整的工作流如下：
+
+```mermaid
+flowchart TD;
+     torch模型-->rewrite_onnx;
+     rewrite_onnx-->ncnn-fp32;
+     rewrite_onnx-->quant_table;
+     quant_table-->ncnn-int8;
+     ncnn-fp32-->ncnn-int8;
+```
+
+mmdeploy 基于静态图（onnx）生成推理框架所需的量化表，再用后端工具把浮点模型转换为定点。
+
+目前 mmdeploy 支持 ncnn PTQ。
+
+## 模型怎么转定点
+
+[mmdeploy 安装](../01-how-to-build/build_from_source.md)完成后，加载 ppq 并安装
+
+```bash
+$ git clone https://github.com/openppl-public/ppq.git
```

Comment:
Can we remove '$'? User might want to copy and paste these command.

Reply:
done.